CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.1.2-DEV] - 2020-12-11
+- fixed sorting in CategoriesMenu module
+- fixed calling urls with Umlaut
+
 ## [1.1.1-DEV] - 2020-12-11
 - Fix missing field and palette problem in contao 4.9 (#9)
```

Comment:
Kannst du bitte hier alles in einer neuen Version 1.1.1 zusammenfassen? Das DEV verdeutlicht dir nur, dass das noch nicht getaggt wurde :-)

Reply:
wurde gemacht :)

Reply:
Tatsächlich gehören Changelogs gar nicht in einem PR ;)